docs/compatibility/compatibility-matrix.rst

```diff
@@ -95,14 +95,14 @@ Use this matrix to view the ROCm compatibility across successive major and minor
       :doc:`HIPIFY <hipify:index>`,17.0.0,17.0.0
       :doc:`ROCdbgapi <rocdbgapi:index>`,0.71.0,0.71.0
       `ROCdebug-Agent <https://github.com/ROCm/rocr_debug_agent>`_,2.0.3,2.0.3
-      :doc:`rocGDB <rocgdb:index>`,14.1.0,13.2.0
-      :doc:`rocProfiler <rocprofiler:profiler_home_page>`,2.0.60100,2.0.0
+      :doc:`rocminfo <rocminfo:index>`,1.0.0,1.0.0
```

Comment:
Alphabetically, shouldn't rocminfo move below ROCm debugger?

Reply:
I mean, I think you're grouping ROC vs ROCm, so I assumed this would be lower with ROCm debugger?

Reply:
I was sorting based on `a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y, z, [space]`